app/views/pages/privacy.html.erb

```diff
@@ -0,0 +1,105 @@
+    <p class="govuk-body">This privacy notice explains what the Ministry of Justice means when we talk about personal data, why we ask for this information about you and what we do with it when you use this form.</p>
+    <p class="govuk-body">It also explains how we store your data, how you can get a copy of the information we've collected about you and how you can complain if you think we've done something wrong.</p>
+    
+    <h2 class="govuk-heading-m">Who manages this service</h2>
+    <p class="govuk-body">This form is managed by the Ministry of Justice.</p>
+    <p class="govuk-body">The information you submit will be processed by the Ministry of Justice.</p>
+    <p class="govuk-body">The Ministry of Justice is the data controller for the personal data collected by this form.</p>
+    
+    <h2 class="govuk-heading-m">When we ask for personal data</h2>
+    <p class="govuk-body">Whenever we ask for information about you, we promise to:</p>
+    <%= govuk_list ["always let you know why we need it",
+                    "ask for relevant personal information only",
+                    "make sure we do not keep it longer than needed",
+                    "keep your information safe and make sure nobody can access it unless authorised to do so",
+                    "only share your data with other organisations for legitimate purposes",
+                    "consider any request you make to correct or delete your personal data"], type: :bullet%>
+    <p class="govuk-body">We also promise to make it easy for you to:</p>
+    <%= govuk_list ["tell us at any time if you want us to stop storing your personal data",
+                    "make a complaint to the supervisory authority"], type: :bullet%>
+    
+    <h2 class="govuk-heading-m">The personal data we collect</h2>
+    <p class="govuk-body">We only collect personal data that you directly provide with your application.</p>
+    <p class="govuk-body">We only collect the information we need to deliver this service to you. The personal data collected includes:</p>
+    <%= govuk_list ["your name and contact details",
+                    "your date of birth",
+                    "any relevant identifiers (e.g. your Prison number) which may help us to find your records"], type: :bullet%>
+    <p class="govuk-body">We <%= govuk_link_to "use cookies", "http://www.aboutcookies.org.uk/managing-cookies" %> to collect data that tells us about how the service is used, including:</p>
+    <%= govuk_list ["your computer, phone or tablet's IP address",
+                    "the region or town where you are using your computer, phone or tablet",
+                    "the operating system and web browser you use"], type: :bullet%>
+    <p class="govuk-body">This information is not used to identify you personally. You can find out more about what cookies we use and why on our <%= govuk_link_to "cookie page", "https://www.gov.uk/help/cookies" %>.</p>
```

Comment:
There is a current ticket to create a new cookies page (/cookies) so this will need to change. It might be easiest to just do it now.

Reply:
Ah yes - I had a thought about whether it would make sense to add an AC to the Cookies page ticket, to make sure to update this link - what do you think? 

I'm not quite sure what stage the cookie stuff is at/when it will be merged and didn't want this ticket to become dependent on it

Reply:
@vertism I've updated the ACs and requirements for this ticket to include updating this link: https://dsdmoj.atlassian.net/browse/CDPT-2447

Is that good with you?